### ISSUE TITLE
Crash caused from plot setbiome

### ISSUE THREAD

**Issue by SecretlyJealous** [IntellectualSites/FastAsyncWorldEdit#671]
# Bug report for FastAsyncWorldEdit 1.16.3

**[REQUIRED] FastAsyncWorldEdit Configuration Files:** 
config.yml: https://pastebin.com/qbx642wn
config-legacy.yml: https://pastebin.com/E4qmuBMb

**[REQUIRED] FastAsyncWorldEdit Version Number:** 
FastAsyncWorldEdit-1.16-376

**[REQUIRED] Spigot/Paper Version Number:** 
git-Paper-204 (MC: 1.16.3) (Implementing API version 1.16.3-R0.1-SNAPSHOT)

**[REQUIRED] Description of the problem:**
From the crash log it looks like someone did a **/p setbiome plains** which caused the server to freeze and crash.
Crash log: https://pastebin.com/S62m4Dvv
**This is all I got in my console and this is the full crash** if you read the crash log it shows that it is stuck waiting on something related to fawe and setting the biome.

**How to replicate**:
I'm not quite sure, this seems like some sort of edge case scenario

**Plugins being used on the server:**
FastAsyncWorldEdit, LuckPerms, PlotSquared, ProtocolLib, LibsDisguises, SimplePets

**Checklist**:

- [X] I included all information required in the sections above
- [X] I made sure there are no duplicates of this report [(Use Search)](https://github.com/IntellectualSites/FastAsyncWorldEdit/issues?q=is%3Aissue)
- [X] I made sure I am using an up-to-date version of [FastAsyncWorldEdit for 1.16.3](https://ci.athion.net/job/FastAsyncWorldEdit-1.16/)
- [X] I made sure the bug/error is not caused by any other plugin

**Comment by dordsor21** [comment]
Could you try this on the latest version of FAWE as well?

**Comment by SecretlyJealous** [comment]
I've been using a newer build and so far I've not yet encountered this crash again, I can't replicate it with //setbiome
